src/dcommands/remind.js

```diff
@@ -42,31 +42,30 @@ Example: ${example}`);
 
         let channel;
         if (input.c) channel = msg.channel.id;
+        let endUnix = moment().add(duration).unix();
         await bu.events.insert({
             type: 'remind',
             source: channel ? msg.guild.id : msg.author.id,
             user: msg.author.id,
             content: input.undefined.join(' '),
             channel: channel,
             starttime: r.epochTime(moment().unix()),
-            endtime: r.epochTime(moment().add(duration).unix())
+            endtime: r.epochTime(endUnix)
         });
-        await bu.send(msg, `:alarm_clock: Ok! I'll remind you ${channel ? 'here' : 'in a DM'} ${duration.humanize(true)}! :alarm_clock: `);
+        await bu.send(msg, `:alarm_clock: Ok! I'll remind you ${channel ? 'here' : 'in a DM'} <t:${endUnix}:R>! :alarm_clock: `);
     }
 
     async event(args) {
-        let duration = moment.duration(moment() - moment(args.starttime));
-        duration.subtract(duration * 2);
         if (args.channel) {
             bu.send(args.channel, {
-                content: `:alarm_clock: Hi, <@${args.user}>! You asked me to remind you about this ${duration.humanize(true)}:
+                content: `:alarm_clock: Hi, <@${args.user}>! You asked me to remind you about this <t:${endUnix}:R> :
```

Comment:
Please define this constant above `if (args.channel)` so it's defined and so it can be used in the DM message